helm/ingress-controller/templates/controller-deployment.yaml

```diff
@@ -65,6 +72,12 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        {{- if .Values.extraVolumeMounts  }}
+        volumeMounts:
+        {{ toYaml .Values.extraVolumeMounts | nindent 10 }}
+        {{- end }}
+        {{- if .Values.extraVolumes }}
```

Comment:
This looks like it might be out of place?  Should it be on line 95 instead?

Reply:
oh good catch, i guess i did't see this in the snapshot because we only have 1 and are providing the value. I guess maybe we should have atleast a snapshot with no overrides set and a snapshot with all options exercised. I'll add that and fix this